src/Modules/Build.lua

```diff
@@ -973,6 +973,21 @@ function buildMode:SyncLoadouts(reset)
 		end
 	end
 
+	table.sort(filteredList, function(a, b)
+		local aText, aNum = a:match("^(.+) {%d+}$"), tonumber(a:match("{(%d+)}"))
+		local bText, bNum = b:match("^(.+) {%d+}$"), tonumber(b:match("{(%d+)}"))
```

Comment:
```suggestion
		local aText, aNum = a:match("^(.+) {%w+}$"), tonumber(a:match("{(%d+)}"))
		local bText, bNum = b:match("^(.+) {%w+}$"), tonumber(b:match("{(%d+)}"))
```

The identifier doesn't have to be a number and I think we still want to sort by name if it's not